<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pipeline Viewer</string>
  </property>
  <widget class="kwiver::tools::ImageView" name="imageView"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Pipe&amp;line</string>
    </property>
    <addaction name="actionExecutePipeline"/>
    <addaction name="actionSuspendUpdates"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFitToWindow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExecutePipeline"/>
   <addaction name="actionSuspendUpdates"/>
   <addaction name="separator"/>
   <addaction name="actionFitToWindow"/>
  </widget>
  <action name="actionExecutePipeline">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Execute Pipeline File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFitToWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="zoom-fit-best">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Fit to Window</string>
   </property>
  </action>
  <action name="actionSuspendUpdates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playback-pause">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Suspend Updates</string>
   </property>
   <property name="toolTip">
    <string>When active, new data from the pipeline will be ignored. This will effectively &quot;pause&quot; the display.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>kwiver::tools::ImageView</class>
   <extends>QGraphicsView</extends>
   <header>ImageView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
